bodhi/server/services/updates.py

```diff
@@ -536,6 +538,25 @@ def new_update(request):
 
             if len(releases) > 1:
                 result = dict(updates=updates)
+
+            if from_tag:
+                koji = buildsys.get_session()
+                for update in updates:
+                    if update.release.composed_by_bodhi:
```

Comment:
we want to do the following if the release is **not** composed by bodhi.

Reply:
I thought you want to create it if it is composed by bodhi. My mistake, I will fix it.

Reply:
Fixed